packages/mui-material/test/typescript/index.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+
+export default function TestStandardPropsUsage() {
+  const contentRef = React.useRef<HTMLDivElement | null>(null);
+
+  const setContentRef = React.useCallback((node: HTMLDivElement | null) => {
+    contentRef.current = node;
+    // ...
+  }, []);
+
+  return (
+    <Dialog open>
+      <DialogTitle>Dialog Demo</DialogTitle>
+      <DialogContent ref={setContentRef}>
```

Comment:
This looks good. Could you also test the usage of object refs in another test (`<DialogContent ref={contentRef}>`)?